### PR TITLE
adding (temp) logo

### PR DESCRIPTION
**Pull request by Reception123**
I know this is not the definite logo but I think we should have a logo, and we can change it later.

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -122,6 +122,10 @@
             'password' => $wmgSMTPPassword,
         ),
     ),
+    // Logo
+    ),
+    'wgLogo' => array(
+	    'default' => "//$wmgUploadHostname/meta.miraheze.org/d/dc/Miraheze_first_logo.png",
```
Comment (NDKilla) [inline LocalSettings.php]: Upload host name isn't defined?
Also the array isn't closed

File: LocalSettings.php [hunk 2]
```diff
@@ -122,7 +122,12 @@
             'password' => $wmgSMTPPassword,
         ),
     ),
-
+    // Logo
```
Comment (Southparkfan) [inline LocalSettings.php]: There's a "Style" section below for this.

File: LocalSettings.php [hunk 3]
```diff
@@ -122,7 +122,12 @@
...
+    ),
```
Comment (Southparkfan) [inline LocalSettings.php]: Remove this.

File: LocalSettings.php [hunk 4]
```diff
@@ -122,7 +122,12 @@
...
+    'wgLogo' => array(
+	    'default' => "//$wmgUploadHostname/meta.miraheze.org/d/dc/Miraheze_first_logo.png",
+    (
```
Comment (Southparkfan) [inline LocalSettings.php]: Should be "),"

File: LocalSettings.php [hunk 5]
```diff
@@ -122,7 +122,12 @@
...
+$wmgUploadHostname = "static.orain.org";
```
Comment (Southparkfan) [inline LocalSettings.php]: Needs to be before "$wgConf->settings"

File: LocalSettings.php [hunk 6]
```diff
@@ -121,8 +123,10 @@
             'username' => 'noreply',
             'password' => $wmgSMTPPassword,
         ),
-    ),
-
+        
+    'wgLogo' => array(
+	    'default' => "//$wmgUploadHostname/meta.miraheze.org/d/dc/Miraheze_first_logo.png",
+    (
```
Comment (NDKilla) [inline LocalSettings.php]: Needs to be ) not (

File: LocalSettings.php [hunk 7]
```diff
@@ -15,6 +15,8 @@
 if ( !defined( 'MEDIAWIKI' ) ) {
     die( 'Not an entry point.' );
 }
+f
```
Comment (NDKilla) [inline LocalSettings.php]: Why is there an f here

File: LocalSettings.php [hunk 8]
```diff
@@ -120,9 +121,10 @@
             'auth' => false,
             'username' => 'noreply',
             'password' => $wmgSMTPPassword,
-        ),
-    ),
 
+    'wgLogo' => array(
+	    'default' => "//$wmgUploadHostname/meta.miraheze.org/d/dc/Miraheze_first_logo.png",
+    (
```
Comment (NDKilla) [inline LocalSettings.php]: This still needs to be a ) instead of a ( ..

File: LocalSettings.php [hunk 9]
```diff
@@ -120,9 +121,10 @@
             'auth' => false,
             'username' => 'noreply',
             'password' => $wmgSMTPPassword,
-        ),
-    ),
```
Comment (NDKilla) [inline LocalSettings.php]: Both of these removals need to be removed. the SMTP arrays above need to be closed

**Comment by NDKilla** [comment]
Duplicated and fixed by #9 ?

**Pull request closed without merging**